### PR TITLE
Fix XML .tt/.ttinclude files to match sources

### LINKED ISSUE #37365: XML text templates do not generate same files as in the repo
Repro:

```
cd src\libraries\System.Private.Xml\src\System\Xml\Core
TextTransform TextUtf8RawTextWriter.tt
```

now diff shows extra newlines at front and some fields renamed to contain underscore.

I suspect similar issue for other fields.

Found when trying to annotate for nullable.

### PR DESCRIPTION
**Pull request by krwq**
Fixes: https://github.com/dotnet/runtime/issues/37365

This is needed to make nullable annotations easier on those files

### CODE REVIEW

File: src/libraries/System.Private.Xml/src/System/Xml/Core/XmlEncodedRawTextWriter.cs [hunk 1]
```diff
@@ -179,7 +174,7 @@ public XmlEncodedRawTextWriter(Stream stream, XmlWriterSettings settings) : this
 
             _encoder = _encoding.GetEncoder();
 
-            if (!stream.CanSeek || stream.Position == 0)
```
Comment (krwq) [inline src/libraries/System.Private.Xml/src/System/Xml/Core/XmlEncodedRawTextWriter.cs:182]: find & replace fail

File: src/libraries/System.Private.Xml/src/System/Xml/Core/XmlEncodedRawTextWriterAsync.cs [hunk 2]
```diff
@@ -51,7 +51,7 @@ internal override async Task WriteXmlDeclarationAsync(XmlStandalone standalone)
                 }
 
                 // Standalone
-                if (standalone != XmlStandalone.Omit)
```
Comment (krwq) [inline src/libraries/System.Private.Xml/src/System/Xml/Core/XmlEncodedRawTextWriterAsync.cs:54]: same

File: src/libraries/System.Private.Xml/src/System/Xml/Core/XmlUtf8RawTextWriterAsync.cs [hunk 3]
```diff
@@ -50,7 +50,7 @@ internal override async Task WriteXmlDeclarationAsync(XmlStandalone standalone)
                 }
 
                 // Standalone
-                if (standalone != XmlStandalone.Omit)
```
Comment (krwq) [inline src/libraries/System.Private.Xml/src/System/Xml/Core/XmlUtf8RawTextWriterAsync.cs:53]: same

File: src/libraries/System.Private.Xml/src/System/Xml/Core/XmlRawTextWriterGeneratorAsync.ttinclude [hunk 4]
```diff
@@ -53,7 +52,7 @@ namespace System.Xml
                 }
 
                 // Standalone
-                if (_standalone != XmlStandalone.Omit)
+                if (standalone != XmlStandalone.Omit)
```
Comment (krwq) [inline src/libraries/System.Private.Xml/src/System/Xml/Core/XmlRawTextWriterGeneratorAsync.ttinclude:55]: ok, seems that was someone before me changed this

File: src/libraries/System.Private.Xml/src/System/Xml/Core/XmlRawTextWriterGenerator.ttinclude [hunk 5]
```diff
@@ -759,8 +755,8 @@ namespace System.Xml
                             _stream = null;
                         }
                     }
-                }
-<# if (WriterType == RawTextWriterType.Encoded) { #>
+                }<#
+if (WriterType == RawTextWriterType.Encoded) { #>
```
Comment (buyaa-n) [inline src/libraries/System.Private.Xml/src/System/Xml/Core/XmlRawTextWriterGenerator.ttinclude:759]: NIT: feels like spacing will not be right on generated files, but i might miss something, not gonna comment about spacing anymore

**Review by buyaa-n — APPROVED** [review]
Left a few questions, overall looks good, thanks!

File: src/libraries/System.Private.Xml/src/System/Xml/Core/RawTextWriter.ttinclude [hunk 6]
```diff
@@ -1,7 +1,6 @@
-﻿<#@ template debug="false" hostspecific="false" language="C#" #>
```
[the remark this answers is not in the document]
Comment (buyaa-n) [reply src/libraries/System.Private.Xml/src/System/Xml/Core/RawTextWriter.ttinclude:1]: I see this removed from *.include files, which seems was redundant as it was already defined in *.tt files

File: src/libraries/System.Private.Xml/src/System/Xml/Core/RawTextWriterUtf8.ttinclude [hunk 7]
```diff
@@ -1,11 +1,9 @@
-<#@ template debug="false" hostspecific="false" language="C#" #>
-<#@ assembly name="System.Core" #>
-<#@ output extension=".cs" #>
```
[the remark this answers is not in the document]
Comment (buyaa-n) [reply src/libraries/System.Private.Xml/src/System/Xml/Core/RawTextWriterUtf8.ttinclude:3]: Same removed from *.include files only which was redundant

**Pull request merged**